react/components/GalleryItem.js

```diff
@@ -49,17 +50,29 @@ const GalleryItem = ({ item, displayMode, summary }) => {
     return normalizedProduct
   }
 
-  return (
-    <ExtensionPoint
-      id="product-summary"
-      {...summary}
-      product={normalizeProductSummary(item)}
-      displayMode={displayMode}
-    />
-  )
+  sendProductClickEvent = product => {
+    this.props.push({
+      event: 'productClick',
+      product: product,
+    })
+  }
+
+  render() {
+    const { item, displayMode, summary } = this.props
+
+    return (
+      <ExtensionPoint
+        id="product-summary"
+        {...summary}
+        product={this.normalizeProductSummary(item)}
+        displayMode={displayMode}
+        actionOnClick={() => this.sendProductClickEvent(item)}
+      />
+    )
+  }
 }
 
-GalleryItem.propTypes = {
+export const propTypes = {
```

Comment:
you removed the prop types from the component.

and why do you need to export this?

Reply:
I removed because Pixel overwritten it anyway, but I exported to make it still available if other components need it.

Reply:
no, the pixel won't override the component's propTypes, it creates a _new_ component that wraps this one, the propTypes are not lost in the way